datajunction-server/datajunction_server/internal/namespaces.py

```diff
@@ -108,3 +115,35 @@ def create_namespace(session: Session, namespace: str):
         ),
     )
     session.commit()
+
+
+def hard_delete_namespace(session: Session, namespace: str, cascade: bool = False):
+    """
+    Hard delete a node namespace.
+    """
+    node_names = session.exec(
```

Comment:
Where do we handle deleting other namespaces under the one being deleted?

Reply:
Good point! I'll add a check for that